.github/workflows/unix.yml

```diff
@@ -85,7 +85,7 @@ jobs:
           - name: arm64
             runner: oqs-arm64
             container: openquantumsafe/ci-ubuntu-focal-arm64:latest
-            PYTEST_ARGS: --numprocesses=auto --maxprocesses=10 --ignore=tests/test_kat_all.py
+            PYTEST_ARGS: --maxprocesses=10 --ignore=tests/test_kat_all.py
```

Comment:
Why did the 'auto' (# of processes started) param get dropped?

Reply:
It's still in effect; I just factored it out into the `python3 -m pytest ...` command below. It was used almost universally, so I figured that there was no point in duplicating it across all jobs.